internal/testblobs/slow.go

```diff
@@ -0,0 +1,104 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package testblobs
+
+import (
+	"context"
+	"sync/atomic"
+	"time"
+
+	"storj.io/storj/storage"
+	"storj.io/storj/storagenode"
+)
+
+// SlowDB implements slow storage node DB.
+type SlowDB struct {
+	storagenode.DB
+	blobs *SlowBlobs
+}
+
+// NewSlowDB creates a new slow storage node DB wrapping apping the provided db.
```

Comment:
"wrapping apping" lol

Reply:
tell me how I've done it :)

Reply:
oh sorry unclear-- the comment probably should just say "DB wrapping the provided db" instead of "wrapping apping the provided db"

Reply:
I totally got it the first time. I wanted to say that I had no idea how I made such a mistake.